app/controllers/admin/videos_controller.rb

```diff
@@ -8,20 +8,36 @@ class Admin::VideosController < Admin::BaseController
   #   video.update(video_params)
   # end
 
-  def create
-    begin
-      tutorial  = Tutorial.find(params[:tutorial_id])
-      thumbnail = YouTube::Video.by_id(new_video_params[:video_id]).thumbnail
-      video     = tutorial.videos.new(new_video_params.merge(thumbnail: thumbnail))
-
-      video.save
+  # def create
+  #   begin
+  #     tutorial  = Tutorial.find(params[:tutorial_id])
+  #     thumbnail = YouTube::Video.by_id(new_video_params[:video_id]).thumbnail
+  #     video     = tutorial.videos.new(new_video_params.merge(thumbnail: thumbnail))
+  #     video.save
+  #
+  #     flash[:success] = "Successfully created video."
+  #   rescue # Sorry about this. We should get more specific instead of swallowing all errors.
```

Comment:
@nagerz This comment to be more specific in errors gave me the idea to refactor and add specific errors to the flash. So, if the video_id is wrong and the title is empty, it will now let the user know both errors.